libs/methodologies/bold/rule-processors/mass-id/avoided-emissions/src/avoided-emissions.errors.ts

```diff
@@ -2,9 +2,15 @@ import { BaseProcessorErrors } from '@carrot-fndn/shared/methodologies/bold/proc
 
 export class AvoidedEmissionsProcessorErrors extends BaseProcessorErrors {
   override readonly ERROR_MESSAGE = {
+    INVALID_MASS_ID_DOCUMENT_SUBTYPE:
+      'The "MassID" document has an invalid subtype.',
+    INVALID_WASTE_GENERATOR_BASELINES:
+      'The "Waste Generator homologation" document has no valid baselines.',
     MISSING_MASS_ID_DOCUMENT: 'The "MassID" document was not found.',
     MISSING_RECYCLER_HOMOLOGATION_DOCUMENT:
       'The "Recycler homologation" document was not found.',
+    MISSING_WASTE_GENERATOR_HOMOLOGATION_DOCUMENT:
+      'The "Waste Generator homologation" document was not found.',
     REJECTED_BY_ERROR: 'Unable to validate the avoided-emissions process.',
```

Comment:
[nitpicking] Considering that these messages are used in several rules, we could have them shared across them all to make sure we are always using the same messages.